services/core-web/src/components/mine/Permit/PermitConditionForm.tsx

```diff
@@ -0,0 +1,275 @@
+                    condition: reportCondition,
+                    permitGuid,
+                },
+                content: ReportPermitRequirementForm,
+            })
+        );
+    };
+
+    const editableProps = !editingConditionGuid && canEditPermitConditions ?
+        {
+            onClick: startEdit,
+            title: "Edit Condition",
+            "aria-label": "Edit Condition",
+        } : {};
+
+    // deals with how the formatting prevents backspace
+    const handleBackSpace = (event, value: string, prev: string, name: string) => {
+        const { nativeEvent } = event;
+        if (nativeEvent?.inputType === "deleteContentBackward" && value === prev) {
+            event.preventDefault();
+            const newVal = value.substring(0, value.length - 1);
+            dispatch(change(formName, name, newVal))
+        }
+    };
+
+    return (
+        <FormWrapper
+            isEditMode={isEditMode}
+            onSubmit={handleSubmit}
+            name={formName}
+            initialValues={condition}
+            scrollOnToggleEdit={false}
+            reduxFormConfig={{
+                enableReinitialize: true
+            }}
+        >
+            <Row
+                wrap={false}
+                className={`condition-content ${!editingConditionGuid ? "editable" : ""}`}
+            >
+                <Col className="step-column" style={{ flexShrink: 0 }}>
+                    <Field
+                        format={(value: string) => formatPermitConditionStep(value)}
+                        parse={(value: string) => parsePermitConditionStep(value)}
+                        name="step"
+                        component={RenderField}
+                        showNA={false}
+                        disabled={isAddingListItem}
+                        onChange={handleBackSpace}
+                    />
+                </Col>
+                <Col className="condition-column"
+                    {...editableProps}
+                >
+                    <Field
+                        name="condition"
+                        component={RenderAutoSizeField}
+                        disabled={isAddingListItem}
+                    />
+                </Col>
+            </Row>
+            {isEditMode && !isAddingListItem && (
+                <Row justify="space-between" align="middle">
+                    <Col>
+                        <Row gutter={8}
+                            className="condition-edit-buttons"
+                        >
+                            <Col>
+                                <Button
+                                    className="fa-icon-container btn-sm-padding"
+                                    type="default"
+                                    icon={<FontAwesomeIcon icon={faPlus} />}
+                                    onClick={handleAddListItem}
+                                >List Item</Button>
+                            </Col>
+                            {/* <Col>
```

Comment:
Just checking but is this suppose to be removed?

Reply:
Well, it's a button that's not implemented yet. Someone (Mat I think) had already put it in and made it look nice, and we are _planning_ on implementing it, so I didn't want to _completely_ remove it, but I also want to hide it because it doesn't do anything.